include/frontend/assembler/assembler.hpp

```diff
@@ -142,17 +142,17 @@ class Assembler final {
         return chai::utils::instr2RawRI(op, func_id, func_id);
     }
     chai::bytecode_t processN(chai::interpreter::Operation op) {
-        return chai::utils::instr2Raw(op, 0, 0);
+        return chai::utils::instr2RawRR(op, 0, 0);
```

Comment:
@levBagryansky replace it by `instr2RawN` pls

Reply:
@c71n93 done